LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Ibiyemi Abiodun
```

Comment:
This should be updated to reflect everyone in the group contributing to this project

Reply:
Sorry, that file was generated automatically and I forgot to update it.